be/src/fs/fs_starlet.cpp

```diff
@@ -34,38 +36,23 @@ using staros::starlet::fslib::kS3AccessKeySecret;
 using staros::starlet::fslib::kS3OverrideEndpoint;
 using staros::starlet::fslib::kSysRoot;
 
-static Status to_status(absl::Status absl_status) {
```

Comment:
why introduce two kinds of status?

Reply:
starlet library using absl::Status, which is not the same as starrocks::Status, so a conversion is needed.

Reply:
But why use another `absl::Status`, does it has any advantage ?

Reply:
Well, I believe starrocks::Status is brought from an earlier version of absl::Status and is extended with some customized error codes. Starlet/StarOS is a new project which doesn't want to couple with starrocks project at the beginning. starlet project depends on grpc which depends on absl, so we decide to use absl thirdparty library, including its mutex/string/status/statusor libraries.

The bottom line is, StarletFileSystem will not leak this dependency to upper level. So the user who uses StarletFileSystem will not see absl Status, and starrocks::Status is still the canonical Status in starrocks project.